kolibri/locale/ach_UG/LC_MESSAGES/kolibri.core.default_frontend-messages.json

```diff
@@ -12,6 +12,13 @@
   "AppBar.openNav": "crwdns334491:0crwdne334491:0",
   "AppBar.pointsAriaLabel": "crwdns369159:0crwdne369159:0",
   "AppBar.pointsMessage": "crwdns369161:0points={points}crwdne369161:0",
+  "AppError.defaultErrorExitPrompt": "crwdns369995:0crwdne369995:0",
+  "AppError.defaultErrorHeader": "crwdns369997:0crwdne369997:0",
+  "AppError.defaultErrorMessage": "crwdns369999:0crwdne369999:0",
+  "AppError.defaultErrorReportPrompt": "crwdns370001:0crwdne370001:0",
+  "AppError.defaultErrorResolution": "crwdns370003:0crwdne370003:0",
+  "AppError.resourceNotFoundHeader": "crwdns370005:0crwdne370005:0",
+  "AppError.resourceNotFoundMessage": "crwdns370007:0crwdne370007:0",
```

Comment:
just out of curiosity, do these map our component string identifier to an identifier on Crowdin's side?

Reply:
Yes, I think that's what these ultimately map to. The text that forms the values here are picked up by the Crowdin in context translation Javascript code and then substituted for the current translation for that string. As seen here: https://translate.learningequality.org/